integration/src/adjudication/mappers/loyalty.mapper.ts

```diff
@@ -32,11 +39,29 @@ export class LoyaltyMapper {
           result.balances.current ? result.balances.current + acc : acc,
         0,
       );
-      basketCredits.offers = this.getCreditOffers(
+      processedCreditOffers = this.getCreditOffers(
         basketCreditResults,
         accounts,
       );
+      basketCredits.offers = processedCreditOffers.offers;
+    }
+
+    const isQuestAccountPresent = accounts.find(
```

Comment:
I would have moved this and the following if statement inside the calculateQuestCampaignProgress method.

Reply:
small refactoring can be done later

Reply:
Treating this as a low priority comment and considering the certification goal, not fixing this in this PR.